x-pack/plugins/endpoint/public/applications/endpoint/store/hosts/selectors.ts

```diff
@@ -88,6 +88,11 @@ export const policyResponseActions: (
   }
 );
 
+export const policyResponseLoading = (state: Immutable<HostState>): boolean =>
+  state.policyResponseLoading;
+
+export const policyResponseError = (state: Immutable<HostState>) => state.policyResponseError;
```

Comment:
Should add explicit return type:

```suggestion
export const policyResponseError = (state: Immutable<HostState>): boolean => state.policyResponseError;
```